### ISSUE TITLE
"expected call-signature: '...' to have a typedef" on setters

### ISSUE THREAD

**Issue by nolazybits** [palantir/tslint#394]
I received the following error
expected call-signature: 'accessToken' to have a typedef
when trying to have a setter like below

```ts
	set accessToken(value: string)
	{
		// some code, no return value as it is a setter
	}
```

which is what typescript expects.

**Comment by adidahiya** [comment]
which version of tslint are you using?

**Comment by nolazybits** [comment]
Ooops, I forgot to mention this didn't I :/ 
We are using tslint 2.1.1

The workaround so far is 
```
/* tslint:disable:typedef */
set mysetter(value: number)
{
/* tslint:disable:typedef */
...
```
Thanks for looking at it

**Comment by nolazybits** [comment]
Please also note that we are using tslint via the grunt task grunt-tslint

**Comment by adidahiya** [comment]
Ok, thanks for the info. We should be releasing new versions of tslint and grunt-tslint soon, although that might not happen until TypeScript 1.5 is out of beta.

**Comment by dbeckwith** [comment]
I am having this same issue as well.

**Comment by dbeckwith** [comment]
Was this fixed in latest release?

**Comment by adidahiya** [comment]
Yes, it should be fixed in v2.3.0-beta. We'll be updating the grunt & eclipse plugins to use that version very soon.

**Comment by dbeckwith** [comment]
Great thanks!
